swan-lake/integration-tools/health-tool.md

```diff
@@ -420,5 +421,224 @@ Follow the steps below to use the generated API templates by running the cloned
         }
     ],
     "id":"1"
+   }
+   ```
+
+## CDS template generation
+
+A Ballerina service template can be generated from the given CDS hook definitions. This template will also include basic functionalities such as validation, prefetch, etc., and it will facilitate the developers' implementation of the required connection with the external decision support system to run the CDS server. The generated Ballerina service project will be written into the provided output location.
```

Comment:
```suggestion
A Ballerina service template can be generated from CDS hook definitions. This template will also include basic functionalities such as validation, prefetch, etc., and it will facilitate the developers' implementation of the required connection with the external decision support system to run the CDS server. The generated Ballerina service project will be written into the provided output location.
```